spec/models/user_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let(:user) {
+              User.create(
+                          fullname: 'Daisy',
+                          display_name: 'lildaisy',
+                          email: 'daisy@example.com',
+                          password: 'password',
+                          role: 'default'
+                          )
+  }
+
+  it 'is valid' do
+    expect(user).to be_valid
+  end
+
+  it 'is not valid without a fullname' do
+    user.fullname = nil
+    expect(user).to_not be_valid
+  end
+
+  it 'is not valid without an email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.